docs/core/tools/global-tools.md

```diff
@@ -126,33 +126,47 @@ The .NET Core CLI tries to add the default locations to the PATH environment var
 * On macOS, if you've installed the .NET Core SDK using *.tar.gz* files and not *.pkg*.
 * On Linux, you need to edit the shell environment file to configure the PATH.
 
-## Other CLI commands
+## Global tools in the .NET Core SDK
 
-The .NET Core SDK contains other commands that support .NET Core Global Tools. Use any of the `dotnet tool` commands with one of the following options:
+The .NET Core SDK contains the `dotnet-aspnet-codegenerator` global tool that can be installed. For example, the following command installs the ASP.NET Core scaffolding tool:
```

Comment:
dotnet-aspnet-codegenerator is not included sdk, or part of the NuGetFallbackFolder.